pages/landing.js

```diff
@@ -1,31 +1,53 @@
 import React from 'react';
-import firebase from 'firebase';
-import Card, {
-  CardHeader,
-  CardTitle,
-  CardContent,
-  CardButtonContainer,
-} from '../components/card';
-import { EDIT } from '../constants';
-import Button from '../components/button';
+import styled from 'styled-components';
+import Sidebar from '../components/sidebar';
+import { getHackathons } from '../utility/firebase';
+import nwPlusLogo from '../assets/nwplus.svg';
 
-export default function Landing() {
-  return (
-    <>
-      <div>Login Page</div>
+const Container = styled.div`
+  display: flex;
+  align-items: stretch;
+  min-height: 100vh;
+`;
+
+const NwPlusImage = styled.img`
+  margin-top: 5%;
+  width: 250px;
+  height: 250px;
+  margin-bottom: 2%;
+`;
+
+const WelcomeTitle = styled.h1`
+  font-size: 60px;
+`;
 
-      <Card>
-        <CardHeader>
-          <CardTitle>Hi</CardTitle>
-          <p>Some extra text</p>
-          <CardButtonContainer>
-            <Button type={EDIT}>Hi there</Button>
+const InfoMessage = styled.p`
+  font-size: 40px;
```

Comment:
Nit: same as above comment
```suggestion
  font-size: 32px;
```